### PR TITLE
New Migrate Local

### LINKED ISSUE #2741: New Doc Proposal: Migrate Local Sites
Priority (Low‚ Medium‚ High): Medium

## Title
Migrate Local Sites to Pantheon 

## Issue Description
Some users end up on the more difficult manual migration path when moving a local Drupal site to Pantheon because step 3 of the guided migration path assumes the site is already live somewhere else. 

## Outline
Intro that summarizes guided migration path is best when migrating a local site
Cover exceptions that may require manual migration
Recommend WP users install the plugin on their local site
Provide steps to move a local Drupal site that do not require a custom domain
Troubleshooting

## Expected Audience
D7 and D8 users looking to take a local site live on Pantheon

## Path
`/docs/migrate-local`

## Next Steps
- [ ] Get @pantheon-systems/cse input based on support trends such as common issues/questions, terminology (migrate vs import vs upload?)
- [ ] Strategize doc update with a corresponding dashboard update so step 3 in the guided path links to new doc

### PR DESCRIPTION
**Pull request by eabquina**
Created from Migrate Manual and removed un-related sections

Need to verify "terminus site:import" section copied that says can also import Wordpress archive

Closes #2741 

## Effect
PR includes the following changes:
- New migrate-local doc
- Updated sections
-

## Remaining Work
- [ ] Verify "terminus site:import" use for Wordpress
- [ ] Review and update /docs/local-development context

### CODE REVIEW

**Comment by rachelwhitton** [comment]
@eabquina can you indicate what next steps are required? Curious if this is still in progress or ready for copy edits. Thanks for the PR!

**Comment by eabquina** [comment]
@rachelwhitton i think the site:import use case might be a long shot and will make that in a separate Issue PR. I will mark this ready for copy edits

File: source/_docs/migrate-local.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+tags: [migrate, local]
+categories: []
+---
+
+This page discusses sites developed from your local machine, ready to deploy on Pantheon.
+
+We recommend using [Kalabox](/docs/kalabox) because of its integrated features to the platform. For large teams/sites, we recommend using [Multidev](/docs/multidev) instead of local development.
+Read more on what will only work for [Local Development](/docs/local-development/)
+
+If you're site is existing or can be synced only, use the [Guided Migration method](/docs/migrate). For cases where the guided migration will not work, follow the [manual migration method](/docs/migrate-manual).
```
Comment (alexfornuto) [inline source/_docs/migrate-local.md]: @eabquina what do you mean by:

> ...is existing or can be synced only...

Comment (eabquina) [reply source/_docs/migrate-local.md]: @alexfornuto the sentence is totally wrong and I've entered a change for that. This is meant to ask if its hosted somewhere else

**Comment by alexfornuto** [comment]
@eabquina We've determined that a lot of this new guide is duplicate content in other migration docs. We're aware that our migration docs are becoming unwieldy, and will be consolidated. As part of that process, concerns for local migrations that need to be addressed will be included into existing docs.

I'm keeping this PR open for the moment, until the additional information has been conflated with larger changes.

**Comment by alexfornuto** [comment]
Closing this PR in favor of revisions to existing doc.

**Pull request closed without merging**